### DISCUSSION TITLE
Recurring Transactions vs Bills (to track fixed monthly costs)

### DISCUSSION THREAD

**Discussion by Jonas3110** [firefly-iii/firefly-iii#9470] [General]
Hey Guys :) 

I didn't really use bills yet, rather recurring transactions, but I wonder if I should switch or use both. I like that the bills, contrary to recurring transactions, can be part of a report and that a summary of the bills is shown on the overview page. Therefore bills seem like a better fit to me to track fixed costs? Wjhat do you guys think? 

 What is your setup to handle this? Do you use bills or recurring transactions or both? And why?

**Comment by JC5** [comment]
Although I now import using GoCardless, I used to use both to make sure I caught everything.

**Comment by floriskruisselbrink** [comment]
Bills and recurring transactions serve two different purposes. 
Bills help to get an overview of all the fixed monthly costs, split by already payed and upcoming or overdue.
Recurring transactions are nothing more than a tool to help me get all my real-world transactions into firefly. For my main bank-account I import transactions using GoCardless so I don't need firefly to create (recurring) transactions. But for my creditcard, there is no usable export/import available so I have to enter all the transactions myself. Here I have setup a few recurring transactions in firefly for things that occur on a fixed pattern.

Hope I made myself clear :)
